public/content/developers/docs/apis/json-rpc/index.md

```diff
@@ -387,6 +387,8 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"eth_syncing","params":[],"id":1}
 
 Returns the client coinbase address.
 
+> **Note:** This method has been deprecated as of **v1.14.0** and is no longer supported. Attempting to use this method will result in a "Method not supported" error.
```

Comment:
> Attempting to use this method will result in a "Method not supported" error.

Should we consider replacing this sentence with a comment similar to below about using `eth_accounts` instead? Don't think it needs to block here, but curious thoughts for further iteration.